templates/edit.html

```diff
@@ -6,9 +6,26 @@
   
     $('#remove-karaoke').click(function(e) {
         var new_name = $('#new_file_name').val();
-        new_name = new_name.replace(/\([Kk][Aa][Rr][Aa][Oo][Kk][Ee].*\)/g, "");
-        new_name = new_name.replace(/[Kk][Aa][Rr][Aa][Oo][Kk][Ee]/g, "");
```

Comment:
What is the purpose of the whitespace replacements in line 10 and 15?

Reply:
There were times after removing the words KARAOKE... there were some white spaces left. Just a quick fix.